specs/gloas/beacon-chain.md

```diff
@@ -841,6 +927,13 @@ def get_expected_withdrawals(
     )
     withdrawals.extend(partial_withdrawals)
 
+    # [New in Gloas:EIP7732]
+    # Get builders sweep withdrawals
+    builders_sweep_withdrawals, withdrawal_index, processed_builders_sweep_count = (
+        get_builders_sweep_withdrawals(state, withdrawal_index, epoch, withdrawals)
```

Comment:
im not sure whats the best order here.. , we have:
`get_builder_withdrawals` -> `get_pending_partial_withdrawals` -> `get_builders_sweep_withdrawals` -> `get_validators_sweep_withdrawals`
do we want to do all validators first then builders?